gitstats.py

```diff
@@ -0,0 +1,1015 @@
+#!/usr/bin/env python3
```

Comment:
I just changed this from `python` in the other repo, but now I'm having half a memory that Actions doesn't like the `python3` alias. Maybe it's safer to run the script with `python gitstats.py mathlib docs` in line 23?

Reply:
Sounds good to me. I'll try some test commits to see if things run.

Reply:
Looks like Actions was happy with `python3` after all (it wasn't happy that I originally left off the filename extension though).

I'll push another commit that changes the logic so that the action runs when non-leanprover-community-bot users push, for ease of testing / deployment.